scipy/optimize/_differentialevolution.py

```diff
@@ -1116,6 +1153,29 @@ def solve(self):
 
         return DE_result
 
+    def _result(self, **kwds):
+        # form an intermediate OptimizeResult
+        nit = kwds.get('nit', None)
+        message = kwds.get('message', None)
+        warning_flag = kwds.get('warning_flag', False)
+        result = OptimizeResult(
+            x=self.x,
+            fun=self.population_energies[0],
+            nfev=self._nfev,
+            nit=nit,
+            message=message,
+            success=(warning_flag is not True),
+            population=self._scale_parameters(self.population)
+        )
+        if self._wrapped_constraints:
+            result.constr = [c.violation(result.x) for
+                                c in self._wrapped_constraints]
```

Comment:
Is indentation appropriate here? 
I would have thought
```suggestion
            result.constr = [c.violation(result.x)
                             for c in self._wrapped_constraints]
```
following the reasoning in [PEP8](https://peps.python.org/pep-0008/#should-a-line-break-before-or-after-a-binary-operator) (although this is not exactly a binary operator).

Reply:
(personally I would like use to use black on the entire codebase, which would prevent these nits from occurring.